{"stats": {"errors": 1, "failures": 0, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "index_test:S256Test", "name": "test_address", "time": 0.08088064193725586, "type": "error", "errtype": "builtins.NameError", "message": "name 'prepend' is not defined", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/5AUsK0H7q2C/addresses-from-keys-jsong-pb-sf-072618/test/index_test.py\", line 17, in test_address\n    point.address(compressed=True, testnet=False), mainnet_address)\n  File \"/home/5AUsK0H7q2C/addresses-from-keys-jsong-pb-sf-072618/index.ipynb\", line 82, in address\n    \"    addr = encode_base58(raw)\\n\",\nNameError: name 'prepend' is not defined\n"}]}